Gemfile

```diff
@@ -46,11 +46,13 @@ group :development do
   gem 'spring'
   # Adds spring compatibility to rspec
   gem 'spring-commands-rspec'
-  # Use Pry instead of IRB
-  gem 'pry-rails'
 end
 
 group :development, :test do
+  # Use Pry instead of IRB
+  gem 'pry-rails'
+  gem 'byebug'
```

Comment:
We could use pry-byebug instead which would be integrated with pry. I don't really care though as I doubt I will use this very often.